src/main/java/com/visenze/productsearch/ProductSearchResponse.java

```diff
@@ -150,6 +152,16 @@
     @JsonProperty("query_sys_meta")
     private Map<String, String> querySysMeta;
 
+    @JsonProperty("explanation")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    private Map<String, ViJsonAny> explanation;
```

Comment:
i think we have not confirmed what to put for explanation yet, might as well remove this for now.

Reply:
OK, I'll remove this for now.